packages/vx-shape/src/types/barStack.ts

```diff
@@ -0,0 +1,54 @@
+import { BarGroupBar } from './barGroup';
+import { BaseStackProps, StackKey } from './stack';
+import { PositionScale } from './base';
+
+/**
+ * Each series point j in a stack chart corresponds to the jth element in the input data.
+ * Each point is represented as an array [y0, y1] where y0 is the lower value (baseline) and y1 is the upper value (topline);
+ * the difference between y0 and y1 corresponds to the computed value for this point.
+ *
+ * SeriesPoint is a [number, number] two-element Array with added data and index properties
+ * related to the data element which formed the basis for theSeriesPoint.
+ */
+export interface SeriesPoint<Datum> extends Array<number> {
+  /**
+   * Corresponds to y0, the lower value (baseline).
+   */
+  0: number;
+  /**
+   * Corresponds to y1, the upper value (topline).
+   */
+  1: number;
+  /**
+   * The data element underlying the series point.
+   */
+  data: Datum;
```

Comment:
is it worth adding `index` to this since it's in the comment?

Reply:
This type is copied from `d3-shape`. (I tried to do straight re-export but there was issue.) I think the `index` in the comment means the `0` and `1` fields.